src/index.html

```diff
@@ -21,7 +24,482 @@
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+        />
+      </a>
+      <nav>
+        <ul class="header__navlist">
+          <li class="header__navlist-items">
+            <a
+              href="#"
+              class="is-active header__navlist-linkstyle"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac A1419 on a white background"
+          class="card__img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+          </div>
```

Comment:
The stars rating system is hard-coded to 4 stars. Consider making this dynamic so that it can display different ratings. This could be done using JavaScript or a back-end language. Also, you may consider using a loop to generate the star elements to keep your code DRY (Don't Repeat Yourself).